conf/broker.conf

```diff
@@ -232,6 +232,12 @@ subscriptionKeySharedUseConsistentHashing=true
 # The higher the number, the more equal the assignment of keys to consumers
 subscriptionKeySharedConsistentHashingReplicaPoints=100
 
+# Maximum time in ms for a Analise backlog operation to complete
+subscriptionBacklogScanMaxTimeMs=120000
+
+# Maximum number of entries to be read within a Analise backlog operation
+subscriptionBacklogScanMaxEntries=10000
```

Comment:
Should we make it dynamic or put it to ns/topic policy?

Reply:
this is only a safe guard mechanism, there is no need to make it configurable at underlying layers.

Reply:
Got it.

Reply:
it is important that these two options are still at broker level, because they are there in order to prevent malicious users from breaking the system.

I can follow up add a per namespace values, but I would do it only after some time that users try this feature and provide feedback